### DISCUSSION TITLE
Edit scenes without triggering them

### DISCUSSION THREAD

**Discussion by joaomartelo** [home-assistant/frontend#9354] [Automations / scripts / scenes] [answered]
Hi,

It would be awesome to be able to edit scenes without the need of them to run,

Everytime I edit my night scene , during the day time, all my shutters close! That's really annoying.

Thanks for the great job you guys are doing!

**Comment by frenck** [comment, marked as answer]
That is not a frontend request, but something for the Core.
Feature requests for the core should go into the community forums

Thanks! 👍

**Comment by brystmar** [comment]
I was about to submit this same request.  We have scenes to set (and reset) the lights and white noise machine for our kid's naps and bedtime, so I have to be really careful not to click the "wake up" scene while he's sleeping!

I'll bump this request in the forms too.
